modules/ocf/manifests/serial_getty.pp

```diff
@@ -0,0 +1,8 @@
+class ocf::serial_getty {
+  service { 'serial-getty@ttyS0':
+    ensure   => running,
```

Comment:
is it necessary to `ensure => running`? generally i am not a fan of this pattern. i explained in some other PR, but i think it's a slippery slope that leads to puppet being used as a shitty service supervisor rather than just to manage configuration

i have seen really bad cases where e.g. a server is not ready for use after boot until puppet has run